signingscript/exceptions.py

```diff
@@ -1,23 +1,45 @@
+"""Signingscript exceptions."""
 from scriptworker.exceptions import ScriptWorkerTaskException
 from scriptworker.constants import STATUSES
 
 
 class TaskVerificationError(ScriptWorkerTaskException):
+    """Something went wrong during task verification."""
+
     def __init__(self, msg):
+        """Initialize TaskVerificationError.
```

Comment:
This is from `flake8_docstrings`. I agree that it's too strict, but I like having some automated check to keep the project up to date.  I don't think the exception docstrings hurt.  After this one time hit, we won't be adding a ton of exceptions, so I don't think it should be too high of a maintenance burden.